TypeScript.YAML-tmTheme

```diff
@@ -42,8 +46,12 @@ settings:
 - scope: entity.name.type.enum.ts
   settings: { vsclassificationtype: enum name }
 
-- scope: entity.name.function, entity.name.type, meta.template.expression.ts, variable, entity.other.inherited-class.ts
+- scope: entity.name.type, meta.template.expression.ts, entity.other.inherited-class.ts

```

Comment:
```suggestion
- scope: meta.template.expression.ts, entity.other.inherited-class.ts
```
given entity.name.type is classified as type above .

Reply:
Thanks, good catch. Fixed.